recipes/gromacs/2020/meta.yaml

```diff
@@ -1,16 +1,20 @@
 # Update from dwnload page, e.g. http://manual.gromacs.org/2019.1/download.html
 # For latest release, see
 # http://manual.gromacs.org/documentation/
-{% set name = "gromacs" %}
 {% set version = "2020.4" %}
 {% set md5sum = "aead694ec7d1222cbc40011c88047c3d" %}
+{% set build = 1 %}
+{% set build = build + 100 %}  # [mpi == 'nompi']
```

Comment:
This is commonly done at conda-forge to give Conda's solver a hint to prefer the non-MPI versions if the user didn't request a MPI one explicitly.
(Yes, this is hacky, but Conda currently does not have the functionality to express preferences directly.)